Makefile

```
@@ -50,7 +50,12 @@ gen-sync: do-gen-sync format lint
 .PHONY: test
 ## Run unit and integration tests with pytest
 test:
-	@poetry run pytest
+	@poetry run pytest -m "not local"
+
+.PHONY: test-local
+## Run the integration tests that require Momento Local
+test-local:
```

nit: naming inconsistency with other sdks

```suggestion
test-retry:
```

---

I did that because we named them retry tests because retries were the first thing we did with momento local, and I figured it would be more general in the future. I made a bunch of retry -> momentoLocal name changes in the Java test logic, for example.
I can make it test-retry if we value the consistency though.